db/views/calendar_months_v01.sql

```diff
@@ -0,0 +1,11 @@
+WITH month_dates AS (
+    SELECT date(generate_series) AS month_date
+    FROM generate_series('2018-01-01', now(), '1 month')
+)
+
+SELECT
+    month_date,
+    EXTRACT(MONTH from month_date) as month,
+    EXTRACT(QUARTER from month_date) as quarter,
+    EXTRACT(YEAR FROM month_date) as year
```

Comment:
Would it be useful to put a `YYYY-MM` string into this, for use with other tables?